config/default/translations/messages-en.properties

```diff
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.delivery.babys_condition.baby_repeat.1.baby_details.vomit = Vomits everything
+report.delivery.babys_condition.baby_repeat.1.baby_details.drowsy = Drowsy or unconscious
+report.delivery.babys_condition.baby_repeat.1.baby_details.stiff = Body stiffness
+report.delivery.babys_condition.baby_repeat.1.baby_details.yellow_skin = Yellow skin color
+report.delivery.babys_condition.baby_repeat.1.baby_details.fever = Fever
+report.delivery.babys_condition.baby_repeat.1.baby_details.blue_skin = Blue skin color
+report.delivery.babys_condition.baby_repeat.1.baby_details.r_baby_danger_sign_present = Danger Signs Present
+report.delivery.babys_condition.baby_repeat.1.baby_details.child_doc = Baby uuid
+report.delivery.babys_condition.r_baby_danger_sign_present_joined = Baby Danger signs
+report.delivery.babys_condition.r_baby_danger_sign_present_any = Any Baby Danger Signs Present
+report.delivery.pnc_visits = PNC Visits
+report.delivery.pnc_visits.who_note =  
+report.delivery.pnc_visits.pnc_visits_attended = PNC visits done so far
+report.delivery.pnc_visits.pnc_visits_additional = Additional PNC visits 
+
+report.pregnancy_facility_visit_reminder.facility_visit_reminder = Reminder of Upcoming Health Facility Visit
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.remind_note=
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.remind_method=Client Reminder Method
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations= Custom translations
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_label_translator= Woman Label translation
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_label= Woman label
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_start_label_translator= Woman start label translator
+report.pregnancy_facility_visit_reminder.facility_visit_reminder.custom_translations.custom_woman_start_label= Woman start label
+report.pregnancy_facility_visit_reminder.pregnancy_uuid_ctx = Pregnancy UUID
+report.pregnancy_facility_visit_reminder.visit_date_for_task = Pregnancy Visit Task Date
+report.pregnancy_facility_visit_reminder.data=
+report.pregnancy_facility_visit_reminder.data.__how_reminded=How the health worker reminded the woman, in person or by phone.
+report.pregnancy_facility_visit_reminder.data.meta=
+report.pregnancy_facility_visit_reminder.data.__patient_uuid=Patient UUID
+report.pregnancy_facility_visit_reminder.data.__patient_id=Patient ID
+report.pregnancy_facility_visit_reminder.data.__household_uuid=Household uuid
+report.pregnancy_facility_visit_reminder.data.__source=Source
+report.pregnancy_facility_visit_reminder.data.__source_id=Source ID 
+report.pregnancy_facility_visit_reminder.data.__pregnancy_uuid=Pregnancy UUID
```

Comment:
I think we are meant to be hiding all data fields, so don't need the labels:

![image](https://user-images.githubusercontent.com/789512/68145870-bd285580-ff04-11e9-8a3d-c59ef6e72a5c.png)